CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # UNRELEASED
 
+-   **BREAKING** Add support for experimental encrypted state events, introduce
+    `EncryptionSettings::encrypt_state_events`, `RoomSettings::encrypt_state_events`,
+    and `OlmMachine::encrypt_state_event`.
```

Comment:
This is predominantly a javascript library, so these things have different names as seen by a user of the library (`EncryptionSettings.encryptStateEvents`, etc).

For the same reason: I don't think this is actually a breaking change: as you demonstrate in your test, `new EncryptionSettings()` continues to work just fine.